client-side/src/stories/Select/Select.jsx

```diff
@@ -1,46 +1,54 @@
-import * as React from 'react';
-import Box from '@mui/material/Box';
-import InputLabel from '@mui/material/InputLabel';
-import MenuItem from '@mui/material/MenuItem';
-import FormControl from '@mui/material/FormControl';
+import React from 'react';
+import {Box,InputLabel,MenuItem,FormControl}from '@mui/material';
 import SelectMui from '@mui/material/Select';
```

Comment:
please collect all the imports of material to one

Reply:
כתבתי לך על זה בסלאק

Reply:
I saw what you wrote but I can't understand why? what is the problem ?